manifests/site.pp

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Default to 0:0, 0644 on POSIX, and root, 0644 on Windows
+case $::operatingsystem {
+    'Windows': {
```

Comment:
Let's not set default ownership on Windows. This should be inheritance from the parent directory by default, or set for specific user in the related code. Also note there is no longer root account on any Windows platforms.